src/patterns/dosdp-patterns/nonhuman_disease_series_by_gene.yaml

```diff
@@ -0,0 +1,59 @@
+pattern_name: nonhuman_disease_series_by_gene
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/nonhuman_disease_series_by_gene.yaml
+
+description: 'This pattern is for non-human diseases that are defined based on a variation in one gene, such as many genetic diseases from the Online Mendelian Inheritance in Animals (OMIA).  Examples: TBD '
+
+contributors:
+- https://orcid.org/0000-0002-4142-7153
+- https://orcid.org/0000-0002-5002-8648
+
+classes:
+  disease: MONDO:0005583
+  gene: SO:0001217
+
+relations:
+  has material basis in germline mutation in: RO:0004003
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+
+vars:
+  disease: '''disease'''
+  gene: '''gene'''
+
+name:
+  text: '%s caused by variation in %s'
+  vars:
+  - disease
+  - gene
+
+annotations:
+- annotationProperty: exact_synonym
+  text: '%s %s'
+  vars:
+  - gene
+  - disease
+- annotationProperty: exact_synonym
+  text: '%s related %s'
+  vars:
+  - gene
+  - disease
+- annotationProperty: exact_synonym
+  text: '%s caused by mutation in %s'
+  vars:
+  - disease
+  - gene
+
+def:
+  text: Any %s in which the cause of the disease is a variation in the %s gene.
+  vars:
+  - disease
+  - gene
+
+equivalentTo:
+  text: '%s and ''has material basis in germline mutation in'' some %s'
```

Comment:
@cmungall and @matentzn 
I did not add the taxon here because the taxon is included in the gene record. 
In your opinion, do you think we should add taxon information here? Is there a reason to add this redundancy? 
Thank you!

Reply:
You need either

1. a property chain `'has material basis in germline mutation in' o 'in taxon' -> 'in taxon'`, in RO
2. the ensure the genus is already species-specific. Presumably this is not always the case as some PS analogs will be cross-mammal, cross-vertebrate, etc
3. just make a var for taxon even if it is duplicative with either/both genus/differentia

But going back to your Q from slack about infectious disease classification:

1. I would in general try and keep the ladders parallel. If some order/class/etc level virus is a useful grouping for H, it should also be for NH? But I appreciate this may not always be the case, esp for agents that never infect human, in which case
2. create a `non_human_disease_by_infectious_agent` DP
     - this would only use NH terms as genus
     - the taxon of the host would need to go in as a variable
     - classification would follow NCBITaxonomy
     - we can worry about making the xp-analog relations complete via rules later

If you do go with 2 it points to doing the same this for this PR, e.g. including a taxon var